validate.py

```diff
@@ -163,13 +168,19 @@ def save_views_imgs(lines, size=(500, 500), interactive=False, ext='jpg'):
     # Load file
     print('Loading track file...')
     track = nib.streamlines.load(input_file, lazy_load=True)
+
+    fclass = [f.get('fclass') for f in _SUP_FORMATS if f.get('ext') == ext][0]
```

Comment:
The main reason I added `_SUP_FORMATS` is because it allows storing elements that are related, like the rows in a database. Maybe the name `_SUP_FORMATS` does not describe this behavior, but the purpose of that dict is to map elements to the extension of interest.